contracts/Vault.vy

```diff
@@ -151,12 +149,12 @@ PERMIT_TYPE_HASH: constant(bytes32) = keccak256("Permit(address owner,address sp
 
 
 @external
-def __init__(
+def initialize(
```

Comment:
Create internal method to handle this logic, and then have `__init__` call internal method (while keeping this one for proxy deployment)

Reply:
Blocked by https://github.com/vyperlang/vyper/issues/2251, need a new Vyper release which allows calling internal methods from constructor.